### PR TITLE
bot: Update minimum stake account balance

### PR DESCRIPTION
**Pull request by alex-at-planetariummusic**
This is to deal with the new minimum delegation amount.

@c3pko

### CODE REVIEW

File: bot/src/stake_pool_v0.rs [hunk 1]
```diff
@@ -19,8 +19,9 @@ use {
     },
 };
 
-// Minimum amount of lamports in a stake pool account
-pub const MIN_STAKE_ACCOUNT_BALANCE: u64 = LAMPORTS_PER_SOL;
+// Minimum amount of lamports in a stake pool account. 2282880 is for rent. Without it, we will be
+// below the miniumum delegation amount, and will get InsufficientDelegation errors
+pub const MIN_STAKE_ACCOUNT_BALANCE: u64 = LAMPORTS_PER_SOL + 2282880;
```
Comment (tigarcia) [inline bot/src/stake_pool_v0.rs:24]: Why not make the rent amount a const also?

Comment (mvines) [reply bot/src/stake_pool_v0.rs:24]: better, can we use `RpcClient::get_stake_minimum_delegation()` to fetch the minimum value from the network directly?

Comment (alex-at-planetariummusic) [reply bot/src/stake_pool_v0.rs]: `RpcClient::get_stake_minimum_delegation()` doesn't seem to include rent. Also it's not available in the version of solana-client we're using, and I didn't want to go down the rabbit hole of upgrading that right now. But I've added constants for stake_minimum_delegation and the rent amount.

I _would_ like to know where to find the rent value in the code, though. Right now it's basically a magic number.

Comment (t-nelson) [reply bot/src/stake_pool_v0.rs]: it's not rent, it's the rent-exempt reserve balance.  you can query it via https://docs.rs/solana-client/latest/solana_client/rpc_client/struct.RpcClient.html#method.get_minimum_balance_for_rent_exemption

**Pull request merged**